QUERIES.md

```diff
@@ -40,8 +43,14 @@ REVOKE ALL [PRIVILEGES] ON <db> FROM <user>
 -- revoke all of user's privileges (all DBs and/or cluster admin)
 REVOKE ALL [PRIVILEGES] FROM <user>
 
+-- combine db creation with privilege assignment (user must already exist)
+CREATE DATABASE <name> GRANT <privilege> TO <user>
+CREATE DATABASE <name> REVOKE <privilege> FROM <user>
```

Comment:
I'd much rather be able to revoke privs on creation than grant them. If I'm using auth it's worse to end up with too much access rather than too little.